docs/dev_docs/merger.md

```diff
@@ -0,0 +1,23 @@
+# Preface
+
+*WIP: This document will be updated throughout the project.*
+
+# Merger
+
+This feature was developed to aggregate multiple specification files[[1]](#1-user-docs--formatting-specification-files) into one file.
+
+This can be used to create an overview of e.g. an entire department's tech stack, which can be used for reviews, analysis, or other types of breakdowns.
+
+#### [1] [User Docs: Formatting specification files.](../user_docs/spec_file_format.md)
+
+## Functions
+
+The merger currently has two functions:
```

Comment:
the merger has 3 functions, but i guess getHeader is less relevant than the two others (?). Regardless, it should probably be changed to "The merger has several functions, amongst them are.." or something

Reply:
kinda agree, but then again I'm not completely sure what it does

Reply:
I'm a bit confused as I seem to have overlooked this getHeader function - in the merger.go file, I can find the passed argument header and its usages, and in the main_test.go, I see the header variable.

Can you maybe pinpoint me to this getHeader function? 🥺

Reply:
There was a small continued conversation on discord on this, did it resolve the matter @Haruyzal ?

Reply:
@JonasSkjodt Yes! Please have a look at the update :)